build_web_compilers/lib/src/dev_compiler_builder.dart

```diff
@@ -282,6 +297,17 @@ Future<void> _createDevCompilerModule(
       _fixMetadataSources(
           json as Map<String, dynamic>, scratchSpace.tempDir.uri);
       await buildStep.writeAsString(metadataId, jsonEncode(json));
+
+      // Copy the symbols output, modifying its contents to remove the temp
+      // directory from paths
+      if (generateDebugSymbols) {
+        var symbolsId = module.primarySource
+            .changeExtension(symbolsExtension(soundNullSafety));
+        file = scratchSpace.fileFor(symbolsId);
+        content = await file.readAsString();
+        json = jsonDecode(content);
+        await buildStep.writeAsString(symbolsId, jsonEncode(json));
```

Comment:
Since we don't need to modify the content we can just use `copyOutput`:

```suggestion
        await scratchSpace.copyOutput(symbolsId, buildStep);
```